windows-msi/msi.wxs

```diff
@@ -721,28 +721,28 @@
 
                     <Directory Id="BINDIR" Name="bin">
                         <Component Id="bin.libcrypto_1_1.dll" Guid="{811694A9-DC99-4CFA-BD72-4877F8E32C7C}">
-                            <File Name="libcrypto-1_1$(var.OPENSSL_PLAT).dll" Source="!(bindpath.openvpn)bin\libcrypto-1_1$(var.OPENSSL_PLAT).dll"/>
+                            <File Name="libcrypto-1_1$(var.OPENSSL_PLAT).dll" Source="!(bindpath.openvpn)bin\libcrypto-1_1$(var.OPENSSL_PLAT).dll Checksum="yes""/>
```

Comment:
This line contains a syntax error:

> error CNDL0104 : Not a valid source file; detail: 'yes' is an unexpected token. Expecting white space. Line 724, position 157.

Fix quoting, please.

Reply:
Fixed with 71ca738.

Reply:
@rozmansi @ikreb7 are you sure that `Checksum` should be part of `Source`? I still got the same error. Shouldn't the proper fix be:

```
diff --git a/windows-msi/msi.wxs b/windows-msi/msi.wxs
index 27aabba..6604d73 100644
--- a/windows-msi/msi.wxs
+++ b/windows-msi/msi.wxs
@@ -721,7 +721,7 @@

                     <Directory Id="BINDIR" Name="bin">
                         <Component Id="bin.libcrypto_1_1.dll" Guid="{811694A9-DC99-4CFA-BD72-4877F8E32C7C}">
-                            <File Name="libcrypto-1_1$(var.OPENSSL_PLAT).dll" Source="!(bindpath.openvpn)bin\libcrypto-1_1$(var.OPENSSL_PLAT).dll Checksum="yes"/>
+                            <File Name="libcrypto-1_1$(var.OPENSSL_PLAT).dll" Source="!(bindpath.openvpn)bin\libcrypto-1_1$(var.OPENSSL_PLAT).dll" Checksum="yes"/>
                         </Component>
                         <Component Id="bin.liblzo2_2.dll" Guid="{A1A057F0-235B-4F55-9A3A-CDA2112648F8}">
                             <File Name="liblzo2-2.dll" Source="!(bindpath.openvpn)bin\liblzo2-2.dll" Checksum="yes"/>


```